src/main/java/seedu/address/logic/commands/schedule/ViewTimeTableCommand.java

```diff
@@ -1,26 +1,55 @@
 package seedu.address.logic.commands.schedule;
 
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 
 /**
  * Views all schedules which satisfies selection requirements.
  */
 public class ViewTimeTableCommand extends Command {
-
     public static final String COMMAND_WORD = "tview";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views the Horizontal/Vertical Timetable. \n"
+            + "Format: tview 0 (View Horizontal Timetable)\n"
```

Comment:
Is it better to use v and h instead of 0 and 1?

Reply:
Good idea! Let me change it